crates/pop-cli/src/commands/test/on_runtime_upgrade.rs

```diff
@@ -376,7 +382,7 @@ mod tests {
 				"pop test on-runtime-upgrade --runtime={} --blocktime=6000 \
 			--checks=all --profile=debug -n live --uri={} --at={}",
 				get_mock_runtime(Some(TryRuntime)).to_str().unwrap(),
-				DEFAULT_LIVE_NODE_URL.to_string(),
+				"ws://localhost:9944".to_string(),
```

Comment:
A constant would be nicer :)

Reply:
https://github.com/r0gue-io/pop-cli/pull/582/commits/cc619af503c84a725f7bd79e54bf345c70c610aa